README.md

```diff
@@ -112,19 +112,25 @@ $ ./packages/percolatestudio-migrations/migrate.sh latest --settings ./setting.j
 
   Migrations set a lock when they are migrating, to prevent multiple instances to prevent multiple instances of your clustered app from running migrations simultaneously. If you migrations throw an exception, you will need to manually remove the lock (and ensure your db is still consistent) before re-running the migration. Update the migrations collection like this:
 
-  ``` sh
-  meteor mongo
+  ```
+  $ meteor mongo
```

Comment:
Hopefully the addition of this `$ ` prefix and the extra line break afterward will help clarify for any newbies that the first line is a terminal/Bash shell command whereas the following lines take place in the Mongo shell that opens up thereafter.